Libs/MRML/Core/vtkMRMLNode.h

```diff
@@ -319,6 +319,14 @@ class VTK_MRML_EXPORT vtkMRMLNode : public vtkObject
   vtkSetMacro(Selectable, int);
   vtkBooleanMacro(Selectable, int);
 
+  /// Specifies if the state of this node stored in the scene's undo buffer.
+  /// False by default to make sure that undo can be enabled selectively,
+  /// only for nodes that are prepared to work correctly when saved/restored.
+  /// Nodes with different UndoEnabled value must not reference to each other,
+  /// because restoring states could lead to unresolved node references.
```

Comment:
Does this last sentence conflict with the notes from the commit message:

> Always save the full scene state - saving only state of selected nodes can break node references and make the state of the scene inconsistent. Even if a Slice core module would think that as a result of a user action only specific nodes are updated, due to behavior of custom modules, it is possible that a simple node change triggers events that change state of other nodes, which would not be possible to revert without saving the state of all nodes.

Reply:
In the commit message, it may be sensible to discuss the relationship with UndoEnabled node property.